src/types/Theme.ts

```diff
@@ -92,9 +92,11 @@ export interface Theme {
   loadingIcon: string
 
   // Background
+  backgroundGradientColors: string[]
+  backgroundGradientStart: { x: number; y: number }
+  backgroundGradientEnd: { x: number; y: number }
```

Comment:
Why not keep this in the `background:` field as:

```js
color?: string // maybe not optional
gradient?: {
  colors: string[]
  start: Point
  end: Point
}
```
Now you can use `background.gradient != null` as the condition for rendering the `MaybeLinearGradient` component.